client/src/partials/purchases/create/templates/actions.tmpl.html

```diff
@@ -0,0 +1,3 @@
+<div style="padding : 5px;" class="text-center">
```

Comment:
If this could use 
```html 
class="ui-grid-cell-contents"
``` 

This provides the padding and assigns overflow rules (overflow : ellipses) although this looks like it's just a single icon.

I think it would be good to generally use this class on most cell templates.